### PR TITLE
Branch verge

### PR DESCRIPTION
**Pull request by mad-anoop**
@SabirAmeen @mithunjohnf @GayathriMohan

### CODE REVIEW

File: verge3.html [hunk 1]
```diff
@@ -0,0 +1,437 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="verge3css.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (GayathriMohan) [inline verge3.html]: use meta tag first, before using script and css link

File: verge3.html [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+</head>
+
+<body>
+    <header class="main_header " id="main_header">
```
Comment (GayathriMohan) [inline verge3.html]: classname and idname should never be same.

File: verge3.html [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+                    <h3>Watch Japan's prime minister morph into Mario for the Olympics closing ceremony </h3> </li>
+            </ul>
+        </section>
+        <section id="trending_now">
+            <p>Trending now</p>
+            <h3>Android 7.0 Nougat review: great, but does it matter?</h3>
+            </section>
+    </header>
+    <section id="container_body">
+        <section class="row1 row">
```
Comment (GayathriMohan) [inline verge3.html:42]: Use meaningful class names. row1 doesn't seem to convey the context.

File: verge3.html [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+            <div class="image_scroll_list" id="scroll_image2">
+                <header>
+                    <h2 class="image_scroll_list_h2"> Twitter now lets any user filter out threats and abuse </h2></header>
+                <p>By Tasha Robinson
+                    <span class="p-comment-count heat2">     14
+                              
+                                                    </span>
+                </p>
+            </div>
+            <div class="image_scroll_list" id="scroll_image1">
```
Comment (GayathriMohan) [inline verge3.html]: id repeated again. First usage of id in line number:65

File: verge3.html [hunk 5]
```diff
@@ -0,0 +1,437 @@
...
+                <header>
+                    <h2 class="image_scroll_list_h2"> Twitter now lets any user filter out threats and abuse </h2></header>
+                <p>By Tasha Robinson
+                    <span class="p-comment-count heat2">     14
+                              
+                                                    </span>
+                </p>
+            </div>
+            <div class="image_scroll_list" id="scroll_image2">
```
Comment (GayathriMohan) [inline verge3.html]: id repetition

File: verge3.html [hunk 6]
```diff
@@ -0,0 +1,437 @@
...
+                <h2 class="image_scroll_list_h2"> Twitter now lets any user filter out threats and abuse </h2>
+                <p>By Tasha Robinson
+                    <span class="p-comment-count heat2">     14
+                              
+                                                    </span>
+                </p>
+            </div>
+        </section>
+        <div class="wrapper_video">
+            <div class="video_header">
```
Comment (GayathriMohan) [inline verge3.html:106]: Try not to use too many nested div.(102-113)
Avoid div in video_header_image, caption1 and caption2.
Use a, h, span tags instead.

File: verge3.html [hunk 7]
```diff
@@ -0,0 +1,437 @@
...
+                <div class="main_video">
+                    <div class="main_video_link_image" data-original="https://cdn2.vox-cdn.com/thumbor/RPdSVOUC9O_2skAaxlmMDoRZU_4=/0x0:1920x1080/1025x577/cdn0.vox-cdn.com/uploads/chorus_image/image/50499197/vrg_1183_timn_thumb_video_01.0.0.jpg" data-imgkey="50499197:*:16" data-dim="992x558">
+                        <img class="video_scroll_elements_list_video_link_image_play" src="https://www.vectorcast.com/sites/default/themes/vectorsoftware/images/transparent-play-button.png">
+                    </div>
+                </div>
+                <section class="video_scroll">
+                    <section class="video_scroll_elements">
+                        <div class="video_scroll_elements_list">
+                            <div class="video_scroll_elements_list_video">
+                                <!-- <a class="video_scroll_elements_list_video_link" href="http://www.theverge.com/video/2016/8/16/12508820/samsung-galaxy-note-7-review" class="m-video-hero__main-link" data-chorus-video-id="99285" data-video-title="This is My Next: Connected Lights" data-placement="hub:hero:playlist1:main">   </a>
```
Comment (GayathriMohan) [inline verge3.html]: don't push comments into git

File: verge3.html [hunk 8]
```diff
@@ -0,0 +1,437 @@
...
+            </div>
+        </div>
+        <section class="animated_vertical_navbar ">
+            <ul>
+                <li>
+                    <a href="#">
+                        <img class="login_signup_image" src="https://cdn0.vox-cdn.com/images/verge/default-avatar.v9899025.gif" alt="Default avatar.v9899025">Log in | Sign up
+                    </a>
+                </li>
+                <li><a>HOME</a></li>
```
Comment (GayathriMohan) [inline verge3.html:178]: href should not be skipped in a tag.

File: verge3.html [hunk 9]
```diff
@@ -0,0 +1,437 @@
...
+                <li><a>CARS</a></li>
+                <li><a>TL;DR</a></li>
+                <li><a>FORUMS</a></li>
+                <li><a>SEARCH</a></li>
+            </ul>
+        </section>
+        <section class="animated_vertical_navbar_shadow">
+        </section>
+        <section class="right_vertical_navbar">
+            <artcle class="right_vertical_navbar_article_one">
```
Comment (GayathriMohan) [inline verge3.html]: check article tag spelling.

File: verge3.html [hunk 10]
```diff
@@ -0,0 +1,437 @@
...
+                    </span>
+                </span>
+                <article class="article_hidden">
+                    <p>Tech support is apparently a lot easier if you're rich and famous. For Barbra Streisand, it's as simple as picking up the phone and calling the CEO of Apple to get a quick fix. The legendary singer and...</p>
+                </article>
+            </div>
+        </div>
+        <section class="entry_slot">
+            <div class="entry_slot_list">
+                <div class="entry_slot_list_image  list_image2">
```
Comment (GayathriMohan) [inline verge3.html]: Too much usage of nested div

File: verge3.html [hunk 11]
```diff
@@ -0,0 +1,437 @@
...
+                                                    </span>
+                </p>
+            </div>
+        </section>
+        <div class="wrapper_video">
+            <div class="video_header">
+                <div class="video_header_image">
+                </div>
+                <div class="video_header_caption1">
+                    <h2><span>Top Home Smart Lights</span>
```
Comment (SabirAmeen) [inline verge3.html]: no need of span tag

File: verge3.html [hunk 12]
```diff
@@ -0,0 +1,437 @@
...
+        </section>
+        <section class="right_vertical_navbar">
+            <artcle class="right_vertical_navbar_article_one">
+                <h3 class="right_vertical_navbar_article_one_H3">August 22</h3>
+            </artcle>
+        </section>
+        <section class="right_vertical_navbar_shadow"></section>
+        <div class="entry_slot">
+            <div class="entry_slot_list">
+                <span class="entry_slot_list_image   list_image1">
```
Comment (GayathriMohan) [inline verge3.html:202]: Use list_image1 as id if it is not repeated anywhere else.

File: verge3.html [hunk 13]
```diff
@@ -0,0 +1,437 @@
...
+                </span>
+                <span class="entry_slot_list_content_wrapper">
+                    <span class="entry_slot_list_content_1">
```
Comment (SabirAmeen) [inline verge3.html]: span is inline and h3 block. no block elements inside inline elements

File: verge3.html [hunk 14]
```diff
@@ -0,0 +1,437 @@
...
+        </div>
+        <section class="entry_slot">
+            <div class="entry_slot_list">
+                <div class="entry_slot_list_image  list_image2">
+                </div>
+                <div class="entry_slot_list_content_wrapper">
+                    <div class="entry_slot_list_content_1">
+                        <h3 class="entry_slot_list_content_1_H3">Barbra Streisand fixes Siri with a call to Tim Cook</h3>
+                    </div>
+                    <div class="entry_slot_list_content_2">
```
Comment (GayathriMohan) [inline verge3.html:226]: Use span

File: verge3.html [hunk 15]
```diff
@@ -0,0 +1,437 @@
...
+            <span class="image_scroll_list_second   scroll2_image3"></span>
+            <span class="image_scroll_list_second   scroll2_image4"></span>
+            <span class="image_scroll_list_second   scroll2_image5"></span>
+        </section>
+        <a class="back_to_top">Back to top</a>
+        <a class="see_news_link" href="#main_header">See All News</a>
+    </section>
+    <footer class="bottom_footer">
+        <section class="footer_child_wrapper">
+            <div class="wrapper_bottom_footer_verge_logo_search_bar">
```
Comment (GayathriMohan) [inline verge3.html:275]: Classname should not be so long. Try to reduce to three separations.

File: verge3.html [hunk 16]
```diff
@@ -0,0 +1,437 @@
...
+                <img class="bottom_footer_verge_logo" src="https://cdn2.vox-cdn.com/uploads/chorus_asset/file/3868564/verge-logo-01.0.png">
+                <form class="search_form">
+                    <input class="search_products">
```
Comment (GayathriMohan) [inline verge3.html]: input tag should have a type attribute.

File: verge3.html [hunk 17]
```diff
@@ -0,0 +1,437 @@
...
+                    <li>Longform</li>
+                    <li>Video</li>
+                    <li>Reviews</li>
+                    <li>Tech</li>
+                    <li>Science</li>
+                    <li>Culture</li>
+                    <li>Cars</li>
+                    <li>Design</li>
+                    <li>Business</li>
+                    <li>US & World</li>
```
Comment (SabirAmeen) [inline verge3.html:293]: use &amp with a ; instead of &

Comment (GayathriMohan) [inline verge3.html:293]: Use '&amp' with semicolon instead of using &.

File: verge3.html [hunk 18]
```diff
@@ -0,0 +1,437 @@
...
+                </ul>
+            </div>
+        </section>
+        <ul class="social_networking_sites_plugins">
+            <li><img src="http://hariiyer.com/wp-content/uploads/2014/04/Facebook-Vector-Icon.png"></li>
+            <li><img src="http://hariiyer.com/wp-content/uploads/2014/04/Facebook-Vector-Icon.png"> </li>
+            <li><img src="http://hariiyer.com/wp-content/uploads/2014/04/Facebook-Vector-Icon.png"></li>
+            <li><img src="http://hariiyer.com/wp-content/uploads/2014/04/Facebook-Vector-Icon.png"> </li>
+        </ul>
+        <section class="wrapper_footer_status_corner_wrapper_footer_caption_copy_rights">
```
Comment (GayathriMohan) [inline verge3.html:321]: Long classnames should be changed

File: verge3.html [hunk 19]
```diff
@@ -0,0 +1,437 @@
...
+            <div class="footer_status_corner">
+                <a href="#">Ethics Statement</a>
+                <a href="#">Community Guidelines</a>
```
Comment (SabirAmeen) [inline verge3.html:324]: use ul tag when defining a list

File: verge3.html [hunk 20]
```diff
@@ -0,0 +1,437 @@
...
+                <div class="copy_rights">
+                    <p>
+                        © 2016 <span>Vox Media</span> , Inc. All rights reserved.
+                </p>
+                </div>
+            </section>
+        </section>
+    </footer>
+    <script>
+    $('.menu_icon').click(function() {
```
Comment (GayathriMohan) [inline verge3.html]: JS and jQuery should be included in separate files rather than in html.

Comment (SabirAmeen) [inline verge3.html]: write js in a separate file

File: verge3.html [hunk 21]
```diff
@@ -0,0 +1,437 @@
...
+        $('.animated_vertical_navbar ').removeClass('transform_left');
+        $('.animated_vertical_navbar_shadow ').removeClass('transform_left_shadow');
+
+        $('.animated_vertical_navbar ').addClass('transform_right');
+        console.log("woooooooo");
```
Comment (GayathriMohan) [inline verge3.html]: No console.log statements should be pushed into git

File: verge3css.css [hunk 22]
```diff
@@ -0,0 +1,1163 @@
+body {
+    margin: 0;
```
Comment (SabirAmeen) [inline verge3css.css:2]: use reset.css to reset the margin and padding.

File: verge3css.css [hunk 23]
```diff
@@ -0,0 +1,1163 @@
...
+    font-size: 24px;
+    line-height: 60px;
+    background-color: #fa4b2a;
+    float: left;
+}
+
+#company_logo_image {
+    width: 135.7px;
+    height: 26px;
+    z-index: 10;
```
Comment (GayathriMohan) [inline verge3css.css:30]: Use smaller z-index values

File: verge3css.css [hunk 24]
```diff
@@ -0,0 +1,1163 @@
...
+    height: 60px;
+    float: left;
+}
+
+#new_articles {
+    height: 60px;
+    width: 79px;
+    float: right;
+    text-transform: uppercase;
+    border-left-color: white;
```
Comment (GayathriMohan) [inline verge3css.css:45]: Use color codes

File: verge3.html [hunk 25]
```diff
@@ -0,0 +1,437 @@
...
+                </div>
+            </section>
+        </section>
+    </footer>
+    <script>
+    $('.menu_icon').click(function() {
+        $('.animated_vertical_navbar ').removeClass('transform_left');
+        $('.animated_vertical_navbar_shadow ').removeClass('transform_left_shadow');
+
+        $('.animated_vertical_navbar ').addClass('transform_right');
```
Comment (SabirAmeen) [inline verge3.html]: use a common variable for $('.animated_variable-nav-bar'). use variables for all such occurences

File: verge3css.css [hunk 26]
```diff
@@ -0,0 +1,1163 @@
...
+    display: none;
+    position: relative;
+}
+
+#trending_now p {
+    font-size: 0.8rem;
+    font-weight: bold;
+    color: #fa4b2a;
+    text-transform: uppercase;
+    margin-top: -13px;
```
Comment (GayathriMohan) [inline verge3css.css]: No negative values.

File: verge3css.css [hunk 27]
```diff
@@ -0,0 +1,1163 @@
...
+    margin: auto;
+    color: #fa4b2a;
+}
+
+.drop_down_list {
+    display: none;
+    z-index: 5;
+    position: absolute;
+    top: 70%;
+    left: -1152%;
```
Comment (GayathriMohan) [inline verge3css.css]: 100% is the maximum.

File: verge3css.css [hunk 28]
```diff
@@ -0,0 +1,1163 @@
...
+    height: 225px;
+    list-style-type: none;
+    background-color: #f1f1f1;
+    padding: 0;
+}
+
+.drop_down_list li {
+    display: block;
+    text-decoration: none;
+    border-bottom: 1px solid #cacaca;
```
Comment (GayathriMohan) [inline verge3css.css:130]: Use shorthand property.

File: verge3css.css [hunk 29]
```diff
@@ -0,0 +1,1163 @@
...
+    font-size: 61%;
+}
+
+
+/*IMAGE TILES              IMAGE TILES          IMAGE TILES              IMAGE TILES              IMAGE TILES        IMAGE TILES     IMAGE TILES*/
+
+.row {
+    width: 100%;
+    height: 175px;
+    margin-bottom: 0;
```
Comment (GayathriMohan) [inline verge3css.css:152]: Use reset.css and avoid padding:0 , margin:0 in all the elements.

File: verge3css.css [hunk 30]
```diff
@@ -0,0 +1,1163 @@
...
+    -webkit-font-smoothing: antialiased;
+    line-height: .7;
+    position: absolute;
+    right: 7%;
+}
+
+.p-comment-count:after {
+    content: '';
+    display: block;
+    width: 0;
```
Comment (GayathriMohan) [inline verge3css.css:215]: width and height : 0 is not needed.

File: verge3css.css [hunk 31]
```diff
@@ -0,0 +1,1163 @@
...
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    content: '';
+    background-color: #292484;
+    background-image: linear-gradient(135deg, #292484, #a83279);
+}
+
+.image_scroll {
```
Comment (GayathriMohan) [inline verge3css.css:272]: Use chaining to write the properties.

File: verge3css.css [hunk 32]
```diff
@@ -0,0 +1,1163 @@
...
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    content: '';
+}
+
+.main_video_link_image img {
+    position: absolute;
+    size: 50px;
```
Comment (GayathriMohan) [inline verge3css.css:484]: Size?

File: verge3css.css [hunk 33]
```diff
@@ -0,0 +1,1163 @@
...
+    display: none;
+    top: 36%;
+    left: 8%;
+    width: 50%;
+    height: auto;
+    margin-left: 30px;
+}
+
+.article_hidden p {
+    display: none;
```
Comment (GayathriMohan) [inline verge3css.css:711]: Give a common class to manage the display:none property.

File: verge3css.css [hunk 34]
```diff
@@ -0,0 +1,1163 @@
...
+    right: -100%;
+    background-color: green;
+}
+
+.right_vertical_navbar_transform_left {
+    transition: all 0.5s ease;
+    right: 0px;
+}
+
+.right_vertical_navbar_shadow_transform_left {
```
Comment (GayathriMohan) [inline verge3css.css]: Repetition of code. Check line 987

File: verge3css.css [hunk 35]
```diff
@@ -0,0 +1,1163 @@
...
+    transition: all 0.5s ease;
+    right: -100%;
+}
+
+@media only screen and (min-width: 700px) {
+    .row1 {
+        width: 50%;
+        height: 280px;
+        top: 0px;
+        left: 0px;
```
Comment (GayathriMohan) [inline verge3css.css:1029]: No need of px.
